build-config/resources/Ballerina.toml

```
@@ -2,7 +2,7 @@
 distribution = "2201.10.0"
 org = "ballerinax"
 name = "hubspot.marketing.campaigns"
-version = "@toml.version@"
+version = "1.0.0"
 license = ["Apache-2.0"]
 authors = ["Ballerina"]
 keywords = [] # TODO: Add keywords
```

We have to update the keywords in this file:
```suggestion
keywords = ["hubspot", "crm", "marketing", "campaigns"]
```

---

done